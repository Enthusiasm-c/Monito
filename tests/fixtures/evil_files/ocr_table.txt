
OCR TEXT EXTRACTION RESULT
=========================

Исходное изображение: product_list.jpg
Качество OCR: 87%
Дата обработки: 2024-01-15

ИЗВЛЕЧЕННЫЙ ТЕКСТ:
-----------------

ПРАЙС-ЛИСТ (OCR)

товар:           iPhone14Pro
цена:            99999руб
категория:       смартфоны
статус:          наличии

товар:           SamsungGalaxyS23
цена:            89999
категория:       телефоны
статус:          заказ

товар:           macbookpro14
цена:            200000р
категория:       ноутбуки
статус:          наличии

товар:           ipadair
цена:            59999руб
категория:       планшеты
статус:          склад

КОНЕЦ ТАБЛИЦЫ

ПРИМЕЧАНИЯ OCR:
- Склеенные слова (характерно для OCR)
- Пропущенные пробелы
- Возможные ошибки распознавания
- Неточности в числах
- Смешанные единицы валют (руб/р)

Точность извлечения: 4 из 4 товаров распознаны
Рекомендация: Проверить цены вручную
        