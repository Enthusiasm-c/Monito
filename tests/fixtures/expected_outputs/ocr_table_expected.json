{
  "filename": "ocr_table.txt",
  "expected_rows_in": 4,
  "expected_rows_out": 4,
  "format": "mock_ocr",
  "challenges": [
    "OCR text extraction",
    "Joined words",
    "Text parsing",
    "Format recognition",
    "OCR errors"
  ]
}